docs/source/release_notes.rst

```diff
@@ -6,6 +6,44 @@
 Releases
 ========
 
+
+Version 1.3.0
+====================
+
+Notes
+-----
+
+* Update recommended
+
+  * type() was replaced by isinstance() in the code for gen utilities since python ansible-core 3.12 deprecreated type() funtion.
```

Comment:
There are two typos here:
`funtion` -> `function`
`deprecreated` -> `deprecated`.

Also there might be a better way to phrase this. How about something like this?

> `type()` was replaced by `isinstance()` in the code for gen utilities since ansible-core 2.16 supports Python 3.12 and that version deprecated type() function.